CONTRIBUTING.rst

```diff
@@ -100,3 +100,43 @@ and run ``make requirements``.
 To upgrade all requirements to their latest allowed version you can run
 ``make upgrade-requirements``—this usually results in a large number of upgrades
 and is not something we can merge easily, but it is sometimes a useful exploratory first step.
+
+PR labels
+---------
+
+Product labels
+~~~~~~~~~~~~~~
+All PRs must be tagged with one of the following PR labels:
```

Comment:
Might be worth noting that the reviewer will add the product label for externally submitted PRs, since non-members can't do it.